.github/workflows/performance.yml

```diff
@@ -0,0 +1,36 @@
+name: Performance Tests
+
+on:
+  push:
+    branches:
+      - trunk
+      - 'feature/**'
+
+jobs:
```

Comment:
The other workflows include some inline docs here that outline all the steps that are being run. Can you add something like that here as well?

Reply:
Thanks for pointing it out. Before submitting the PR, I noted this but missed it. I have added inline docs.